TWLight/emails/templates/emails/access_code_email-body-text.html

```diff
@@ -0,0 +1,5 @@
+Your approved application to {{ partner }} has now been finalised. Your access code or login details can be found below:
+
+{{ access_code }}
+
+{{ body }}
```

Comment:
I feel like this should probably be `access_code_instructions` instead of `body` for legibility.
```suggestion
{{ access_code_instructions }}
```